src/jit/morph.cpp

```diff
@@ -12576,8 +12577,10 @@ GenTreePtr Compiler::fgMorphSmpOp(GenTreePtr tree, MorphAddrContext* mac)
 
                     return tree;
                 }
+#endif // LEGACY_BACKEND
                 break;
 
+#ifdef LEGACY_BACKEND
             case GT_QMARK:
```

Comment:
Is this really dead for RyuJIT? We still have qmarks from importer until the end of morph when we call `fgExpandQmarkNodes()`. Are there any diffs?